_test/test_sqw_class/sqw_tester.m

```diff
@@ -0,0 +1,15 @@
+classdef sqw_tester<sqw
```

Comment:
this looks incomplete - it's not a runtest test. Could we have a comment at the top to say where it will be used and why

Reply:
all tester classes are usually to test protected properties of a class. No harm of adding this to the definition.